hack/dind-cluster.sh

```diff
@@ -712,6 +712,7 @@ function copy-runtime() {
   local origin_root=$1
   local target=$2
 
+  cp "$(os::util::find::built_binary openshift-sdn)" "${target}/openshift-sdn-plugin"
```

Comment:
why do we need to rename this?

Reply:
@danwinship because the actual CNI plugin itself is copied to ${target}/openshift-sdn a couple lines below.  One of them needs to be changed...